Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1246,17 +1269,6 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     ASSERT(m_backgroundWebView);
 
     // FIXME: <https://webkit.org/b/246483> Don't unload the background page if there are open ports, pending website requests, or if an inspector window is open.
-
-    if (m_backgroundWebView.get()._isBeingInspected) {
```

Comment:
This code was causing an infinite loop so I removed it. Should be revisited in https://webkit.org/b/246483.